### PR TITLE
Add unit tests for github `GetPullRequestFiles`

### PR DESCRIPTION
**Pull request by shay2025**
## Description

This pull request introduces unit tests for github `GetPullRequestComments` function.

## Related issue

*None*

## Type of change

Improvements (non-breaking change without functionality)

## How was this tested?

Ran `task test` command.

## Checklist

- [x] I have performed a self-review of my code
- [x] I have ran `task check -f` and have no issues

### CODE REVIEW

**Comment by shay2025** [comment]
There is missing the test for when the we are requesting the list of files in the pull request.
This request won't be able to be mocked because the creation of a new env involves getting the list of files in the pull request.
If we really want to add a unit test for this scenario, we'll have to rethink a different approach in the creation of a new env.

**Comment by reviewpad-team** [comment]
**Reviewpad Report**

**:bangbang: Errors**
* **Unconventional commit detected**: 'Add unit tests for github GetPullRequestFiles' (35d4ff094d5fa7f576294939cb10af2527dee978)

File: utils/github_test.go [hunk 1]
```diff
@@ -51,3 +54,42 @@ func TestGetPullRequestNumber(t *testing.T) {
...
+	mockedPullRequestRepoName := mockedPullRequest.Base.Repo.GetName()
+	mockedPullRequestNumber := mockedPullRequest.GetNumber()
+
+	gotFiles, err := utils.GetPullRequestFiles(
+		mockedEnv.GetCtx(),
+		mockedEnv.GetClient(),
+		mockedPullRequestOwner,
+		mockedPullRequestRepoName,
+		mockedPullRequestNumber,
+	)
```
Comment (ferreiratiago) [inline utils/github_test.go:91]: Suggestion. Place the `mocked` values inline.

```go
gotFiles, err := utils.GetPullRequestFiles(
		mockedEnv.GetCtx(),
		mockedEnv.GetClient(),
		mockedPullRequest.Base.Repo.Owner.GetLogin(),
		mockedPullRequest.Base.Repo.GetName(),
		mockedPullRequest.GetNumber(),
)
```

**Review by ferreiratiago — APPROVED** [review]
Minor suggestion.

**Pull request merged**